engine/src/cmd/ai/aggressive.cpp

```diff
@@ -260,16 +260,16 @@ void AggressiveAI::SetParent(Unit *parent1) {
     last_directive = "b";     //prevent escort race condition
 
     //INIT stored stuff
-    Fshield_prev = parent->graphicOptions.InWarp ? 1 : parent->FShieldData();
+    Fshield_prev = parent->ftl_drive.Enabled() ? 1 : parent->FShieldData();
```

Comment:
I wonder if `ftl_drive.IsOn()`, `ftl_drive.TurnOn()`, and `ftl_drive.TurnOff()` might make better sense to the reader?

Reply:
`IsOn`/... is more engine specific. `Enable`/`Disable`/`Enabled` is generally used in programming. 
The choice between the two isn't dramatic. 
Honestly, I like `Enabled` better because it's easier for me to scan quickly in code. I think `IsOn` is harder for me because the capital O breaks the reading flow. 
But, it's marginal and a simple search and replace. So I'll leave it to you to decide.

Reply:
enabled/disabled is sufficient. We want a quick status.